docker-compose.yml

```diff
@@ -105,15 +106,16 @@ services:
     proxy:
         image: nginx:1.15-alpine
         ports:
-            - 80:80
-            - 443:443
+        - 80:80
+        - 443:443
         volumes:
-            - certs-vol:/etc/nginx/certs/
-            - keys-vol:/etc/nginx/keys/
-            - ./volume/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
-        restart: unless-stopped   
-        depends_on:
-            - certs             
+        - ./volume/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+        - /home/greg/workspace10/OpenELIS/dev/https/greg.dev.openelis.org.crt.pem:/etc/nginx/serverCerts/cert.crt
+        - /home/greg/workspace10/OpenELIS/dev/https/greg.dev.openelis.org.key.pem:/etc/nginx/serverCerts/cert.key
+        - /home/greg/workspace10/OpenELIS/dev/https/passwd:/etc/nginx/private/key_pass
+        extra_hosts:
+        - host.docker.internal:172.20.1.1
+        restart: unless-stopped            
```

Comment:
@abertnamanya i gues changes int this docker-compose file were meant for your local dev env.
can you discard them ??

Reply:
Thank you @mozzy11, Please check the changes